### PR TITLE
Converted text encodings from various Japanese to UTF-8

### PR DESCRIPTION
**Pull request by gingerbeardman**
- some were EUC-JP
- some were Shift-JIS
- some were unknown
- all bar one (see below) are now UTF-8

Exception: I believe that the file `menu_str_sjis.txt` needs to be Shift-JIS for this to build. So I have left the file untouched and instead created an extra file `menu_str_utf8.txt` to test that out.

I do not currently have a build system set up, so please check this builds before merging.

### CODE REVIEW

File: fmgen/file.cpp [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #include "file.h"
 
 // ---------------------------------------------------------------------------
-//	����/����
+//	πΩ√€/æ√Ã«
```
Comment (negativeExponent) [inline fmgen/file.cpp]: this did not convert well (and so are some below)

File: fmgen/psg.cpp [hunk 2]
```diff
@@ -12,7 +12,7 @@
 #include "psg.h"
 
 // ---------------------------------------------------------------------------
-//	•≥•Û•π•»•È•Ø•ø°¶•«•π•»•È•Ø•ø
+//	コンストラクタ・デストラクタ
```
Comment (negativeExponent) [inline fmgen/psg.cpp]: these did not convert well too (some others down below)

**Comment by gingerbeardman** [comment]
@negativeExponent those few were fixed in https://github.com/libretro/px68k-libretro/pull/99/commits/37370b88323b745e4542c61b64343b154d19d18d

**Comment by negativeExponent** [comment]
then these should be ok then. just convert the rest if there are any along the way

**Comment by gingerbeardman** [comment]
I think we're good, I converted them all via command line and then checked and fixed any rogues.

**Comment by negativeExponent** [comment]
i didn't notice this before. only source files/headers should have been converted. the rest left alone like .txt files and the sjs.txt for the menu (since there is already a separate file for this)

**Comment by twinaphex** [comment]
Hi @gingerbeardman,

got any idea if the resultant text encodings would still be compilable with older compilers like MSVC 2003 and older GCC versions like on Power PC Mac (circa 4.x ish ) ?

**Comment by gingerbeardman** [comment]
@negativeExponent I left the menu text file unchanged, see PR message. 

@twinaphex then the question is whether those handle files encoded as UTF-8 (Without BOM)? I believe they do.

**Comment by gingerbeardman** [comment]
If we don't think this is needed I'm happy for it to be closed without merge.

Thoughts @negativeExponent ?

**Comment by negativeExponent** [comment]
> If we don't think this is needed I'm happy for it to be closed without merge.
> 
> Thoughts @negativeExponent ?

everyone's busy preparing for next RetroArch release. if you are able to release log that this japanese text in utf8 compiles fine then that would probably make reviewing faster.  if it does not, probably just leave encoding (i think most are in EUC-JP and 8859-1?) at stock until all text on a file can be translated.

i dont have any last say in these though.

**Comment by gingerbeardman** [comment]
gonna close this, can redo at some point if it's really needed.

**Comment by negativeExponent** [comment]
i wanted to reopen and continue this conversion discussion, but repo is already gone.
in anycase, looking at quasi88, the source files are in utf8 while maintaining the japanese text. I believe the original sourcefiles/source encodings are the same with px68k. is there a way to test compile for mvsc targets? travis probably? or the new CI whatever is used now?

**Comment by gingerbeardman** [comment]
Possibly, but that is outside of my area.

Still happy to actually help with, or do, the conversion and translation after compiler queries are answered.

**Pull request closed without merging**